### ISSUE TITLE
Fullscreen I turned it off but it turns on by itself

### ISSUE THREAD

**Issue by svox07** [UnamSanctam/SilentCryptoMiner#1156]
![IMG_20220913_083513_709](https://user-images.githubusercontent.com/70852343/189795044-65ba25a4-cbb3-4d9a-aca9-2d39283d8c28.jpg)
![IMG_20220913_083511_830](https://user-images.githubusercontent.com/70852343/189795048-bead3a8e-be7e-481f-9a18-fc75094bf519.jpg)
![IMG_20220913_083509_054](https://user-images.githubusercontent.com/70852343/189795051-2a224ee7-ef54-4101-803f-3fff77731b44.jpg)
![Screenshot_20220913-080705](https://user-images.githubusercontent.com/70852343/189795054-cd9eafb4-e94c-4503-8218-8e7931e56525.jpg)
![Screenshot_20220912-213338](https://user-images.githubusercontent.com/70852343/189795056-ecc58c54-0bf7-4548-9242-26426780df35.jpg)
pastebin.com configured and the miner disabled this function why is this happening

**Comment by UnamSanctam** [comment]
Maybe the API "stealth-fullscreen" is bugged for XMR, so it's enabled because you enabled it in the builder (on those running miner). I can't check if the "stealth-fullscreen" is bugged (only the API call to change it and not anything else) but it could be likely.

**Comment by svox07** [comment]
when assembling the miner, I do not turn it on, I have never used this function

**Comment by UnamSanctam** [comment]
Maybe it's accidentally taking the option from somewhere else (not impossible since it's a huge list) but then `"stealth-fullscreen": false` should work fine. If you paste your pastebin configuration into a JSON validator does it say that it's valid?

**Comment by svox07** [comment]
I did as you advised "stealth-fullscreen": set Pastebin to false but fullscreen shows in the panel I wanted to clarify the text position is correct stoyt ?
![Uploading IMG_20220913_083511_830.jpg…]()

**Comment by UnamSanctam** [comment]
Yes but verify that your JSON is valid since otherwise it can't load the JSON, there are many sites online for checking or you can check by using the "Configurations" in the web panel since they only accept valid JSON.

**Comment by svox07** [comment]
can you tell me how to check this json

**Comment by UnamSanctam** [comment]
Either paste it into a JSON validator online, or paste it into the "Configurations" tab in the web panel.

**Comment by svox07** [comment]
![IMG_20220913_123649_690](https://user-images.githubusercontent.com/70852343/189828178-345ffcf1-d132-4b85-8ace-3f51bb7e0db7.jpg)
![IMG_20220913_123651_427](https://user-images.githubusercontent.com/70852343/189828183-cbf4c563-f8d2-4de0-ae4a-717426ad7e62.jpg)
checked not skolka online json checks

**Comment by UnamSanctam** [comment]
Yes it looks like your JSON is incorrect, you need all the "kill-targets" to be inside the quotes "" on a single line. Like so `"kill-targets": "target1,target2,target3"` while it looks like you have targets outside of the quotes.

**Comment by svox07** [comment]
Thank you unam for your help.
